components/navbar/ProfileDropdown.vue

```diff
@@ -102,3 +103,15 @@ const toggleLanguageMenu = () => {
   languageDropdown.value?.toggle()
 }
 </script>
+
+<style lang="scss" scoped>
+.menu-mt-0 {
```

Comment:
have you tried `rootClass` or `triggerClass`?

Reply:
I'll check. thanks

Reply:
maybe I am misunderstanding you

I want to affect the menu padding
and here is the menuClass
![image](https://github.com/kodadot/nft-gallery/assets/22791238/04f68804-1a2c-491b-9d93-d9012d036024)

Reply:
got you
done
https://github.com/kodadot/nft-gallery/pull/7460/commits/ae7a21a7d4a3985a8fc985d1790865fd47b62e3a